docs/user_guide/index.md

```diff
@@ -0,0 +1,23 @@
+# User guide
+
+Thanks for trying out JupyterGIS!
+
+```{cssclass} try-in-jupyterlite-button
+
+[Try it with JupyterLite!](lite/)
+```
+
+JupyterGIS offers:
+
+- **Collaborative GIS Environment**: Work together on geographic data projects in real-time.
+- **QGIS File Support**: Load, visualize, and manipulate QGIS project files (`.qgs`, `.qgz`), and other GIS data formats.
+- **Interactive Maps**: Render interactive maps and geospatial visualizations within Jupyter notebooks using the JupyterGIS Python API.
+
+```{toctree}
+:maxdepth: 2
+
+install
+tutorials/index
+features/index
```

Comment:
I'm not convinced by that `features` part.
I wonder if we could not have ``extension` and `collaboration` at top level.

Reply:
My thought process for this change was that this content (`extension` and `collaboration`) fall under the category of "[explanation](https://diataxis.fr/explanation/)", and the audience is end-users. For that reason I thought (1) these things definitely belong in the user guide; (2) it made sense to group these two things together in a directory where we can continue adding more explanatory content in the future. 

What do you think about my reasoning?

Alternatively, what about grouping these documents under a "tour" or "About > Why JGIS?" structure? Some of the wordy content on the document index may also feel at home in a "About" section.